lib/webengine.cpp

```diff
@@ -16,7 +16,27 @@
 
 Q_GLOBAL_STATIC(SailfishOS::WebEngine, webEngineInstance)
 
-void SailfishOS::WebEngine::initialize(const QString &profilePath)
+/*!
+  \class SailfishOS::WebEngine
+  \brief Provides access to the Web engine context.
+  \inmodule SailfishWebView
+
+  Singleton class which provides access to the Web engine context.
+*/
+
+namespace SailfishOS {
```

Comment:
No no... Sailfish

Reply:
Shit why on earth this was like this at the first place.

Reply:
I can go through and change all instances, but it will affect the library C++ API, which I know is in use by external projects.

Reply:
Let's keep this in mind and consider a bit later. I'm quite confident that we just need to break this. Good thing is that it's simple to fix if it's used.